src/Chainweb/Pact/SPV.hs

```diff
@@ -119,7 +119,7 @@ getTxIdx bdb pdb bh th = do
     -- get BlockPayloadHash
     ph <- fmap (fmap _blockPayloadHash)
         $ entries bdb Nothing (Just 1) (Just $ int bh) Nothing S.head_ >>= \case
-            Nothing -> spvError "unable to find payload associated with transaction hash"
+            Nothing -> return $ Left "unable to find payload associated with transaction hash"
             Just x -> return $ Right x
```

Comment:
This is `pure . note "unable to find payload associated with transaction hash"`

Reply:
do we have `errors` as a dependency?

Reply:
Almost certainly.

Reply:
Why is `pure . note` better than `return $ Left`? 

The former might be cooler but the latter seems more readable, in particular for Haskell newcomers to me. And the conciseness argument doesn't apply either.

Reply:
`note` collapses the entire pattern match away.

Reply:
oh,  sorry, I missed that it saves the case distinction. Well, still not convinced that it's worth it, but I now see the benefit.

Reply:
It's `note :: a -> Maybe b -> Either a b`, sibling to `hush :: Either a b -> Maybe b`, which I'd argue every Haskeller has written themselves in some `Utils` module :p 

I think they're handy to have around in general.